Dockerfile

```diff
@@ -79,21 +79,31 @@ RUN \
     install_clean libdvd-pkg && \
     dpkg-reconfigure libdvd-pkg
 
-
-###########################################################
-# Final image pushed for use
-FROM deps-ripper as arm-dependencies
-
 # install makemkv and handbrake
-RUN install_clean \
-        handbrake-cli \
-        makemkv-bin \
-        makemkv-oss
-
+#RUN apt update && install_clean handbrake-cli
+#COPY ./scripts/install_handbrake.sh /install_handbrake.sh
+#RUN chmod +x /install_handbrake.sh && sleep 1 && \
+#    /install_handbrake.sh
+
+# MakeMKV setup by https://github.com/tianon
+COPY ./scripts/install_makemkv.sh /install_makemkv.sh
+RUN chmod +x /install_makemkv.sh && sleep 1 && \
+    /install_makemkv.sh
 # clean up apt
 RUN apt clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Container healthcheck
+COPY scripts/healthcheck.sh /healthcheck.sh
+RUN chmod +x /healthcheck.sh
+HEALTHCHECK --interval=5m --timeout=15s --start-period=30s CMD /healthcheck.sh
```

Comment:
Copy healthcheck and run it every 5 mins

Allows arm to show unhealthy if anything is broken inside the container